docs/aws-professional/services.md

```diff
@@ -51,12 +51,12 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS profe
-|               Backup               |                                     None                                      |                                                                                                                   [Azure Backup](https://azure.microsoft.com/services/backup/)                                                                                                                   | Backup and archival solutions allow files and folders to be backed up and recovered from the cloud, and provide off-site protection against data loss. There are two components of backup—the software service that orchestrates backup/retrieval and the underlying backup storage infrastructure. |
+|       Archiving &mdash; cool storage       |                           S3 Infrequent Access (IA)                           |                                                                                                          [Azure Storage &mdash; Standard Cool](/azure/storage/blobs/storage-blob-storage-tiers)                                                                                                          |                                                                                                  Cool storage is a lower cost tier for storing data that is infrequently accessed and long-lived.                                                                                                   |
+|       Archiving &mdash; cold storage       |                                  S3 Glacier                                   |                                                                                                        [Azure Storage-Standard Archive](/azure/storage/blobs/storage-blob-storage-tiers)                                                                                                         |                                                                                            Archive storage has the lowest storage cost and higher data retrieval costs compared to hot and cool storage.                                                                                            |
+|               Backup               |                                     None                                      |                                                                                                                   [Azure Backup](https://azure.microsoft.com/services/backup/)                                                                                                                   | Backup and archival solutions allow files and folders to be backed up and recovered from the cloud, and provide offsite protection against data loss. There are two components of backup &mdash; the software service that orchestrates backup/retrieval and the underlying backup storage infrastructure. |
```

Comment:
I think a recent PR creates a conflict here - we added 'AWS Backup'

Reply:
Reverted to version in master.